src/AlignmentVault.sol

```diff
@@ -743,5 +779,4 @@ contract AlignmentVault is Ownable, Initializable, ERC721Holder, ERC1155Holder,
     }
 
     receive() external payable virtual {}
-    fallback() external payable virtual {}
```

Comment:
On second thought, leave the fallback in. There may be a need for the contract to not revert if a bridging solution is depositing something into an AV from Layer 2 or otherwise. In the event such an action would revert if a missing callback were not present (and thus threw the revert), assets could be lost.